src/components/Map.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useEffect, useMemo } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
import React, { useRef, useEffect, useMemo } from "react";
```